### ISSUE TITLE
Regression for platform get

### ISSUE THREAD

**Issue by GoZOo** [platformsh/platformsh-cli#235]
I have regression using platform CLI 1.8.2.

I work on platform with only one environment that is not master.

I want to get this platform in local doing
```shell
platform get xxxxxxx
```

With previous CLI version (like 1.0.3, yes, it's old...), CLI let me choose which environment to choose displaying list.

With current version, error message is 
```shell
Cloning into 'xxxxxxx/repository'...
fatal: Remote branch master not found in upstream origin
Failed to clone Git repository
Please check your SSH credentials or contact Platform.sh support
```

Old functionality was helpful

**Comment by pjcdawkins** [comment]
Yes that seems to be a bug - if you only have one environment, it assumes it's called 'master'. It only offers you a choice if there is more than one.

**Comment by GoZOo** [comment]
In fact, i only have one environment because i'm not the owner and have only rights on this sub-environment.

**Comment by pjcdawkins** [comment]
Thanks - fixed in v1.8.3
